eolearn/visualization/eoexecutor.py

```diff
@@ -223,3 +236,21 @@ def _format_datetime(value: dt.datetime) -> str:
     def _format_timedelta(value1: dt.datetime, value2: dt.datetime) -> str:
         """Method for formatting time delta into report"""
         return str(value2 - value1)
+
+
+@dataclass()
+class _ErrorSummary:
```

Comment:
Is it a standard to put dataclasses to the bottom of the files?

Reply:
we dont have any standards for this. Since it's a private class I didnt really bother with placement and added it on the bottom.